scripts/build.ts

```diff
@@ -20,102 +25,172 @@ function genChainMetadataMapExport(data, exportName) {
 ${genJsExport(data, exportName)} as ChainMap<ChainMetadata>`;
 }
 
-console.log('Preparing tmp directory');
```

Comment:
The diff in this file is a lot less than it seems. I just wrapped the chunks in functions. Use the PR "Hide Whitspace" setting for a better view.

Reply:
thanks for the tip!